.gitlab-ci.yml

```diff
@@ -31,10 +31,11 @@ cache: &cache
   key: $CI_COMMIT_REF_SLUG-$CI_PROJECT_DIR
   paths:
     - node_modules/
+    - tmp/.htmltest/
 
 # ================== templates ================== #
 .base_template: &base_template
-  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/corp-ci:master
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/corp-ci:nsollecito_htmltest
```

Comment:
Is this correct for merging to master?

Reply:
@ruthnaebeck I pushed out a new version of the corp-ci build image and pointed this back to master, so we should be good here. Thanks!